configuration/NI.Analysis.targets

```diff
@@ -0,0 +1,95 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <CodeAnalysisRuleSetDirectory>$(PkgNI_DotNet_Analyzers)\content</CodeAnalysisRuleSetDirectory>
```

Comment:
Necessary?

Reply:
This variable is defined through a <GeneratePathProperty> but I am removing this file for now